WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
```

Comment:
Could you elaborate a bit more on why this is needed? I changed it to an `@ObservedObject` and everything looked the same but chances are that I'm not looking in the correct places 😅

Reply:
I did it because of [this](https://a8c.slack.com/archives/C6H8C3G23/p1619086783178800), but since I also did a massive refactoring, eliminating unnecessary classes and sharing the same view model, `EnvironmentObject` is no longer necessary. 👍 543a74a93f19a32d25a29c8e038b4139c7bdf958